spark2.3/src/main/scala/splice/SpliceDataSource.scala

```diff
@@ -1,6 +1,6 @@
 package splice
 
-import com.splicemachine.spark2.splicemachine.SplicemachineContext
+import _root_.com.splicemachine.spark2.splicemachine.SplicemachineContext
```

Comment:
Why is this needed? It seemed working fine till now.

Reply:
sbt started giving some errors like 

`splice-machine-spark-connector/spark2.3/src/main/scala/splice/SpliceDataSource.scala:3:12: object splicemachine is not a member of package splice.com`
`[error] import com.splicemachine.spark2.splicemachine.SplicemachineContext`

apparently as a result of new shading in jars in the spliceengine project.

Reply:
Do you have the commit perhaps? It'd be nice to have a look (and reference back for the future).

Reply:
Sure, please see https://github.com/splicemachine/spliceengine/commit/56fa7d67695bd73fedebaee59acbeae94fd61e93#diff-939ad65c915560fb7f81cf325a7a2520f74277f01ccd76661b3da46d0e993265